R/parameters.R

```diff
@@ -1,3 +1,48 @@
+##' Create a linear time-varying beta for input into 
+##' \code{create_parameters()}
+##' 
+##' @title Generate beta
+##' 
+##' @param beta_start Value of beta before intervention
+##' 
+##' @param start_date Start date for simualtions
+##' 
+##' @param reduction_start Start of reduction in beta
+##' 
+##' @param beta_reduction Factor by which beta is reduced
+##'   default is 0.238, as estimated by Jarvis et al
+##'   beta_reduced = beta - beta * (1 - beta_reduction)
```

Comment:
simplifies to beta_reduced=beta_reduction? not sure I understand the line

Reply:
This refers to L40:
```
beta_slope <- beta_start * (1 - (1 - beta_reduction ) * (seq(0, reduction_period - 1) / (reduction_period - 1)))
```
Which for b=0.1, reduction - 0.234 over 10 days:
```
0.10000000 0.09148889 0.08297778 0.07446667 0.06595556 0.05744444 0.04893333 0.04042222 0.03191111 0.02340000
```
over 4 days:
```
0.10000000 0.07446667 0.04893333 0.02340000
```
which I think matches with your original